developers/weaviate/client-libraries/python/v3_v4_migration.md

```diff
@@ -134,25 +140,40 @@ import QuickStartCode from '!!raw-loader!/_includes/code/graphql.filters.nearTex
     language="py"
   />
 
+Note that the `Move` import exposes configuration elements for moving `near_text` search towards, or away from, the main query input.
```

Comment:
Consider:
The `Move` import exposes configuration elements for `near_text` search that refine the distance between your query and result vectors.

Reply:
> If it is the former (which I believe should be the case) then we should focus on explaining that you can import each class, and skip the explanation of what Move is used for.

Yup agreed

Reply:
Done